Can't build this repository on Windows.

Rustc: `rustc 1.59.0 (9d1b2106e 2022-02-23)`
Cargo: `cargo 1.59.0 (49d8809dc 2022-02-10)`
NeoVim: `NVIM v0.8.0-dev+296-g307c5c63e`
Error: 
![изображение](https://user-images.githubusercontent.com/53366878/170449978-1fab49b7-0a90-4eb5-83e4-9bf672272e80.png)

Translation of some parts of error message:
ссылка на неразрешённый внешний символ -> link to forbidden external symbol
в функции -> in function
неразрешенных внешних элементов -> forbidden external elements
создается библиотека -> creating a library

FWIW, I'm unable to build `master` on MacOS 12.4 with neovim 0.7.0.:

```
  = note: Undefined symbols for architecture arm64:
            "_nvim_buf_set_lines", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::set_lines::hdd424ae77f2153d5 in nvim_oxi.oxwr7faqjkh86jf.rcgu.o
            "_nvim_buf_get_option", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::get_option::hc21c9e32db803a65 in nvim_oxi.oxwr7faqjkh86jf.rcgu.o
            "_lua_getfield", referenced from:
                nvim_oxi::toplevel::toplevel::print::_$u7b$$u7b$closure$u7d$$u7d$::h192461af31742870 in nvim_oxi.437f80zcb8hopa87.rcgu.o
            "_lua_pushstring", referenced from:
                nvim_oxi::toplevel::toplevel::print::_$u7b$$u7b$closure$u7d$$u7d$::h192461af31742870 in nvim_oxi.437f80zcb8hopa87.rcgu.o
            "_lua_call", referenced from:
                nvim_oxi::toplevel::toplevel::print::_$u7b$$u7b$closure$u7d$$u7d$::h192461af31742870 in nvim_oxi.437f80zcb8hopa87.rcgu.o
            "_lua_pushlstring", referenced from:
                _$LT$T$u20$as$u20$nvim_oxi..lua..lua..LuaPushable$GT$::push::hb8d40955a63a2f77 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_nvim_buf_set_option", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::set_option::h98bab8ca4aa681fb in nvim_oxi.oxwr7faqjkh86jf.rcgu.o
            "_lua_pushboolean", referenced from:
                _$LT$T$u20$as$u20$nvim_oxi..lua..lua..LuaPushable$GT$::push::hb8d40955a63a2f77 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_lua_newuserdata", referenced from:
                nvim_oxi::lua::lua::once_to_luaref::_$u7b$$u7b$closure$u7d$$u7d$::hfd28961c531f93e9 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_nvim_buf_get_var", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::get_var::he819f2a686c150cd in nvim_oxi.oxwr7faqjkh86jf.rcgu.o
            "_lua_pushinteger", referenced from:
                _$LT$T$u20$as$u20$nvim_oxi..lua..lua..LuaPushable$GT$::push::hb8d40955a63a2f77 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_lua_pushnil", referenced from:
                _$LT$T$u20$as$u20$nvim_oxi..lua..lua..LuaPushable$GT$::push::hb8d40955a63a2f77 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_lua_pushnumber", referenced from:
                _$LT$T$u20$as$u20$nvim_oxi..lua..lua..LuaPushable$GT$::push::hb8d40955a63a2f77 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_luaL_ref", referenced from:
                nvim_oxi::lua::lua::once_to_luaref::_$u7b$$u7b$closure$u7d$$u7d$::hfd28961c531f93e9 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_nvim_buf_call", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::call::he260fd1adc497b3f in nvim_oxi.oxwr7faqjkh86jf.rcgu.o
            "_lua_touserdata", referenced from:
                nvim_oxi::lua::lua::once_to_luaref::c_fun::h08797fa95de8af92 in nvim_oxi.1n6u1fw4hsiiqv27.rcgu.o
            "_nvim_get_current_buf", referenced from:
                nvim_oxi::api::global::global::get_current_buf::h4521663ad3b8687f in nvim_oxi.3ivdcb1ywicb5r0o.rcgu.o
            "_nvim_create_buf", referenced from:
                nvim_oxi::api::global::global::create_buf::h8eda9da327b87f79 in nvim_oxi.3ivdcb1ywicb5r0o.rcgu.o
            "_nvim_get_mode", referenced from:
                nvim_oxi::api::global::global::get_mode::h1ab1b1788af7b16d in nvim_oxi.3ivdcb1ywicb5r0o.rcgu.o
            "_nvim_buf_set_var", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::set_var::hcbd15dcb6f0d8b10 in nvim_oxi.oxwr7faqjkh86jf.rcgu.o
            "_lua_pushcclosure", referenced from:
                nvim_oxi::lua::lua::once_to_luaref::_$u7b$$u7b$closure$u7d$$u7d$::hfd28961c531f93e9 in nvim_oxi.3sn6kj4i5c0x2x71.rcgu.o
            "_nvim_buf_get_name", referenced from:
                nvim_oxi::api::buffer::buffer::Buffer::get_name::h5ebd92b2dd787a52 in nvim_oxi.17x8afrijgrogobk.rcgu.o
          ld: symbol(s) not found for architecture arm64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)


warning: `nvim-oxi` (lib) generated 94 warnings
error: could not compile `nvim-oxi` due to previous error; 94 warnings emitted
```

--- a/nvim-types/src/string.rs
+++ b/nvim-types/src/string.rs
@@ -1,6 +1,9 @@
 use std::borrow::Cow;
 use std::ffi::OsStr;
+#[cfg(target_family = "unix")]
 use std::os::unix::ffi::OsStrExt;
+#[cfg(target_family = "windows")]
+use std::os::windows::ffi::OsStrExt;
 use std::path::PathBuf;
 use std::string::{self, String as StdString};
 use std::{fmt, slice, str};

